src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
 body {
   margin: 0;
 }
+
+.card {
+  max-width: 200px;
+  height: 406px;
+
+  box-sizing: border-box;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  position: relative;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__picture {
+  position: absolute;
+  top: 33px;
+
+  max-width: 160px;
+
+}
+
+.card__item-model {
+  position: absolute;
+  top: 207px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  width: 85%;
+}
+
+.card__item-code {
+  position: absolute;
+  top: 247px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: rgba(97, 96, 112, 1);
+  text-align: left;
+  width: 85%;
+
+}
+
+.card__item-price {
+  position: absolute;
+  top: 317px;
+
+  display: flex;
+  justify-content: space-between;
+  width: 85%;
+
+}
+
+.card__item-review {
+  position: absolute;
+  top: 277px;
+
+  display: flex;
+  justify-content: space-between;
+  width: 85%;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  width: 85%;
+}
+
+.stars__star {
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+}
+
+.card__review {
+  margin: 0;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400px;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: rgba(6, 11, 53, 1);
```

Comment:
SCSS Variables are used for main values and placed in a separate file. Colors are main values most of the time. Also create variables if value repeats more than once.